views/show.blade.php

```diff
@@ -84,12 +84,12 @@
 @stop
 
 @section('css')
-{!! Asset::styles('logviewer') !!}
+    {!! HTML::style(asset('styles/logviewer.css') !!}
```

Comment:
The html component is deprecated and doesn't exist in laravel 5.1 at all. That's why I wanted to remove usage of it so I could upgrade to L5.1.